### ISSUE TITLE
Update countries names

### ISSUE THREAD

**Issue by Esteban82** [GenericMappingTools/dcw-gmt#17]
I compare the names of the DCW file with the [wikipedia page](https://en.wikipedia.org/wiki/ISO_3166-1) of the readme. I found the some different names: 

I think that the firsts (up to Wallis) should be updated. Do you agree? 

And what about the rest? The last three are the same words in other order. 

dcw-countries.txt | Wikipedia
-- | --
Antigua | Antigua and Barbuda
Brunei | Brunei Darussalam
Cape Verde | Cabo Verde
Falkland Islands | Falkland Islands (Malvinas)
Mariana | Northern Mariana Islands
Surinam | Suriname
Swalizand | Eswatini
Trinidad | Trinidad and Tobago
Wallis | Wallis and Futuna
Bolivia | Bolivia (Plurinational State of)
Czech Republic | Czechia
Micronesia | Micronesia (Federated States of)
United Kingdom | United Kingdom of Great Britain and Northern Ireland
Iran | Iran (Islamic Republic of)
Moldova | Moldova, Republic of
Palestine | Palestine, State of
Sint Maarten | Sint Maarten (Dutch part)
Syria | Syrian Arab Republic
Turks and Caicos | Turks and Caicos Islands
Taiwan | Taiwan, Province of China
Tanzania | Tanzania, United Republic of
United States | United States of America
Holy See (Vatican City State) | Holy See
Venezuela | Venezuela (Bolivarian Republic of)
British Virgin Islands | Virgin Islands (British)
Virgin Islands | Virgin Islands (U.S.)
Democratic Republic of the Congo | Congo, Democratic Republic of the
Democratic People's Republic of Korea | Korea (Democratic People's Republic of)
Republic of Korea | Korea, Republic of

**Comment by Esteban82** [comment]
@PaulWessel should I include these modifications in dcw 2.0.1?

**Comment by PaulWessel** [comment]
Yes, up to and including Wallis and Futuna is fine, the rest is just long...

**Comment by Esteban82** [comment]
Ok. 

Should I also update ChangeLog? Or is it done automatically?

**Comment by PaulWessel** [comment]
Yes, please do.  Not automatic at all.

**Comment by Esteban82** [comment]
Ok. Done. 

I have the dcw-gmt-2.0.1.tar.gz file. I have to make a release, right? 

Later I will try to see how to do it. Is there any guide?

**Comment by PaulWessel** [comment]
No, I think I have to do it for now.  Question is are there any other glitches we should fix as a bug fix while we are at it?  I think the new "continents" will be 2.1.0 but if we get that done soon then is there a point in releasing 2.0.1 and then 2.1.0 a week later?  I think we just wait for now.  You can test your tar ball by installing the data, set cmake to point to it, and try it.

**Comment by Esteban82** [comment]
I just check the rest of the "countries". They are all ok.  I

So we just have to decide the "continents".

By the way, I change Ukraine from Asia to Europe.

**Comment by Esteban82** [comment]
Al tough, now I am thinking that we could let the new "continents" for [list of oceans, seas and other regions](https://github.com/GenericMappingTools/gmt/issues/5742). So, here we could keep the continents that we already have. We just would need to reassigned some countries (like of the SA to NA).  And call it a bug fixed.

**Comment by Esteban82** [comment]
I will update the data for the provinces of Argentina. I usually use 6 but I see that it is vary. How many decimal numbers I must use? Does this matter? 

```
> canada 0
-65.6469421387	43.4177780151
-65.6466674805	43.4174995422

> italy 0
          8.309679         39.130558
          8.308619         39.110172
          8.305259         39.105526
          8.296204         39.104427
```

**Comment by PaulWessel** [comment]
6 decimals is good since that is about 10 cm.
